lib/rollbar/middleware/js.rb

```diff
@@ -121,6 +121,8 @@ def close_old_response(response)
       end
 
       def config_js_tag(env)
+        require 'json'
```

Comment:
just curious, do we need to keep a require 'json' inside the method? why not with all the other requires?

Reply:
In Ruby, this puts the require only in the scope of that method. (The only method where we call `JSON.generate()`.) This allows the rest of the code to work without the dependency.